SCClassLibrary/Common/Control/Volume.sc

```diff
@@ -6,12 +6,17 @@ Volume {
 
 	var <ampSynth, <numOutputChannels, defName, updateFunc, initFunc;
 	var <>window;
+	var <permID, <baseAmpSynth;
 
 	*new { | server, startBus = 0, numChannels, min = -90, max = 6, persist = false |
 		^super.newCopyArgs(server ?? { Server.default }, startBus, numChannels, min, max, persist).init;
 	}
 
 	init {
+		permID = permID ?? { server.nextPermNodeID };
```

Comment:
this is not a safety check, but a lazy init: it should only be made once. 
in case any code runs the server's init method again, we still want to have the same permID. since this happens on Server:new, we can be sure it is 2, and any permIDs created later will be higher numbers.

Reply:
i don't understand -- "in case any code runs the server's init method again" -- when does that happen where it is not a programming error?

Reply:
you are right, made permID only once.